assignments/week05/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link rel="stylesheet" href="css/style.css" />
+        <title>Assignment 05 - CSS Grid</title>
+    </head>
+    <body>
+<!-- Grid wrapper for all content page-->
+        <main class="main">
+<!-- Adding Header -->
+        <header class="header">
+            Header
+        </header>
+ <!-- Adding Nav bar -->
+        <nav class="nav">
```

Comment:
The nav here would be a good spot to add the BEM, ie `.nav`, `.nav__list`, `.nav__item`.